### DISCUSSION TITLE
show all user

### DISCUSSION THREAD

**Discussion by ManarangID** [munafio/chatify#233] [Q&A] [answered]
hy, how to show all user / contact without using searching method? so we can choose contact to star chat them 
thanks and regards

**Comment by munafio** [comment, marked as answer]
@ManarangID publish the controller then modify it .. use the same logic of `search` method for `getContacts` method, but you have to know what you are doing and understand the code.

**Reply by ManarangID** [reply]
i have no idea to do it. i want to display contacts by id. please help, thank you

**Reply by munafio** [reply]
if you want to show all logged in user contact list, then you don't need to do anything .. because Chatify does it by default, shows all users once you message each other.

but if you want to shows all users of the app, then do the same of what I said above.

**Reply by ManarangID** [reply]
thanks so much, regards
